packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -592,6 +592,8 @@ export class FeatureDevController {
             canSelectFiles: false,
         }).prompt()
 
+        let metricData: MetricData
```

Comment:
Since this `MetricData` type won't be used anywhere else, it's ok to use it inline here:
```
let metricData: { result: 'Succeeded' } | { result: 'Failed'; reason: string } | undefined;
```